Origin: EmberBSD (AI-assisted). Build these consumers with the one Ruby, Go and
MySQL client version in the export instead of a removed version.

--- a/devel/Guitar/Makefile
+++ b/devel/Guitar/Makefile
@@ -11,7 +11,7 @@ HOMEPAGE=	https://github.com/soramimi/Guitar/
 COMMENT=	Git GUI Client
 LICENSE=	gnu-gpl-v2
 
-TOOL_DEPENDS+=	ruby34-[0-9]*:../../lang/ruby34
+TOOL_DEPENDS+=	${RUBY_BASE}>=${RUBY_VERSION}:${RUBY_SRCDIR}
 
 USE_TOOLS+=	pkg-config
 USE_LANGUAGES=	c c++
@@ -32,11 +32,12 @@ do-configure:
 	cd ${WRKSRC}/filetype && ${QTDIR}/bin/qmake liboniguruma.pro
 	cd ${WRKSRC}/filetype && ${SETENV} ${MAKE_ENV} ${MAKE}
 	cd ${WRKSRC} && ${QTDIR}/bin/qmake Guitar.pro
-	${LN} -s ${PREFIX}/bin/ruby34 ${BUILDLINK_DIR}/bin/ruby
+	${LN} -s ${TOOLBASE}/bin/${RUBY_NAME} ${BUILDLINK_DIR}/bin/ruby
 
 post-install:
 	${INSTALL_PROGRAM} ${WRKSRC}/_bin/Guitar ${DESTDIR}${PREFIX}/bin
 
+.include "../../lang/ruby/rubyversion.mk"
 .include "../../graphics/qt6-qtsvg/buildlink3.mk"
 .include "../../x11/qt6-qtbase/buildlink3.mk"
 .include "../../mk/bsd.pkg.mk"
--- a/net/gnunet/buildlink3.mk
+++ b/net/gnunet/buildlink3.mk
@@ -27,7 +27,7 @@ pkgbase:= gnunet
 .endif
 
 .if ${PKG_BUILD_OPTIONS.gnunet:Mmysql}
-.include "../../databases/mysql80-client/buildlink3.mk"
+.include "../../mk/mysql.buildlink3.mk"
 .endif
 
 .if ${PKG_BUILD_OPTIONS.gnunet:Mpgsql}
--- a/net/qrcp/Makefile
+++ b/net/qrcp/Makefile
@@ -16,8 +16,6 @@ LICENSE=	mit
 WRKSRC=		${WRKDIR}/${PKGNAME_NOREV}
 USE_LANGUAGES=	c
 
-TOOL_DEPENDS+=	go118>=1.18.0:../../lang/go118
-
 GO_BUILD_PATTERN=	.
 
 .include "./go-modules.mk"
